module/zfs/dbuf.c

```diff
@@ -613,8 +614,7 @@ dbuf_cache_multilist_index_func(multilist_t *ml, void *obj)
 static inline unsigned long
 dbuf_cache_target_bytes(void)
 {
-	return MIN(dbuf_cache_max_bytes,
-	    arc_target_bytes() >> dbuf_cache_shift);
+	return MIN(dbuf_cache_max_bytes, arc_c_max >> dbuf_cache_shift);
```

Comment:
Why have you changed this? To me previous dbuf sizing based on ARC sizing looked reasonable.  With this you practically remove back-pressure on dbuf cache when OS asks for RAM back.